dbt_platform_helper/domain/maintenance_page.py

```diff
@@ -387,6 +387,12 @@ def create_header_rule(
     )
 
 
+def add_default_cidr_prefix(ip_network: str, default_cidr_prefix_length: int):
+    if "/" in ip_network:
+        return ip_network
+    return ip_network + "/" + str(default_cidr_prefix_length)
```

Comment:
The tests suggest that there is not a default value.

```suggestion
def ensure_cidr_suffix(ip_network: str, cidr_suffix: int):
    if "/" in ip_network:
        return ip_network
     
    return ip_network + "/" + str(cidr_suffix)
```

Reply:
I've removed the prefix parameter (we should never need to default it to anything other than 32) and renamed, hopefully making intention clearer.